perf/data/foaf.ttl

```diff
@@ -0,0 +1,699 @@
+  rdfs:comment "An agent (eg. person, group, software or physical artifact)." ;
+  owl:equivalentClass dc:Agent .
+
+geo:SpatialThing
+  a owl:Class ;
+  rdfs:label "Spatial Thing" .
+
+foaf:Document
+  a rdfs:Class, owl:Class ;
+  rdfs:label "Document" ;
+  rdfs:comment "A document." ;
+  ns0:term_status "stable" ;
+  owl:equivalentClass schema:CreativeWork ;
+  rdfs:isDefinedBy foaf: ;
+  owl:disjointWith foaf:Organization, foaf:Project .
+
+foaf:Organization
+  a rdfs:Class, owl:Class ;
+  rdfs:label "Organization" ;
+  rdfs:comment "An organization." ;
+  ns0:term_status "stable" ;
+  rdfs:subClassOf foaf:Agent ;
+  rdfs:isDefinedBy foaf: ;
+  owl:disjointWith foaf:Person, foaf:Document .
+
+foaf:Group
+  a rdfs:Class, owl:Class ;
+  ns0:term_status "stable" ;
+  rdfs:label "Group" ;
+  rdfs:comment "A class of Agents." ;
+  rdfs:subClassOf foaf:Agent .
+
+foaf:Project
+  a rdfs:Class, owl:Class ;
+  ns0:term_status "testing" ;
+  rdfs:label "Project" ;
+  rdfs:comment "A project (a collective endeavour of some kind)." ;
+  rdfs:isDefinedBy foaf: ;
+  owl:disjointWith foaf:Person, foaf:Document .
+
+foaf:Image
+  a rdfs:Class, owl:Class ;
+  ns0:term_status "stable" ;
+  rdfs:label "Image" ;
+  rdfs:comment "An image." ;
+  owl:equivalentClass schema:ImageObject ;
+  rdfs:subClassOf foaf:Document ;
+  rdfs:isDefinedBy foaf: .
+
+foaf:PersonalProfileDocument
+  a rdfs:Class, owl:Class ;
+  rdfs:label "PersonalProfileDocument" ;
+  rdfs:comment "A personal profile RDF document." ;
+  ns0:term_status "testing" ;
+  rdfs:subClassOf foaf:Document .
+
+foaf:OnlineAccount
+  a rdfs:Class, owl:Class ;
+  ns0:term_status "testing" ;
+  rdfs:label "Online Account" ;
+  rdfs:comment "An online account." ;
+  rdfs:isDefinedBy foaf: ;
+  rdfs:subClassOf owl:Thing .
+
+owl:Thing rdfs:label "Thing" .
+foaf:OnlineGamingAccount
+  a rdfs:Class, owl:Class ;
+  ns0:term_status "unstable" ;
+  rdfs:label "Online Gaming Account" ;
+  rdfs:comment "An online gaming account." ;
+  rdfs:subClassOf foaf:OnlineAccount ;
+  rdfs:isDefinedBy foaf: .
+
+foaf:OnlineEcommerceAccount
+  a rdfs:Class, owl:Class ;
+  ns0:term_status "unstable" ;
+  rdfs:label "Online E-commerce Account" ;
+  rdfs:comment "An online e-commerce account." ;
+  rdfs:subClassOf foaf:OnlineAccount ;
+  rdfs:isDefinedBy foaf: .
+
+foaf:OnlineChatAccount
+  a rdfs:Class, owl:Class ;
+  ns0:term_status "unstable" ;
+  rdfs:label "Online Chat Account" ;
+  rdfs:comment "An online chat account." ;
+  rdfs:subClassOf foaf:OnlineAccount ;
+  rdfs:isDefinedBy foaf: .
+
+foaf:mbox
+  a rdf:Property, owl:InverseFunctionalProperty, owl:ObjectProperty ;
+  ns0:term_status "stable" ;
+  rdfs:label "personal mailbox" ;
+  rdfs:comment "A  personal mailbox, ie. an Internet mailbox associated with exactly one owner, the first owner of this mailbox. This is a 'static inverse functional property', in that  there is (across time and change) at most one individual that ever has any particular value for foaf:mbox." ;
```

Comment:
```suggestion
  rdfs:comment "A personal mailbox, i.e., an Internet mailbox associated with exactly one owner, the first owner of this mailbox. This is a 'static inverse functional property', in that there is (across time and change) at most one individual that ever has any particular value for foaf:mbox." ;
```

Reply:
@TallTed thanks for the comments - these are sample files (intended only for performance testing) pulled directly from https://timbl.inrupt.net/profile/card and http://xmlns.com/foaf/0.1/ respectively - if there are problems with the contents I think it would be better if you followed up with the original sources for these files.